apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -10,6 +10,7 @@ div.pl-order-blocks-bottom {
   /* This particular style renders the dropzones in a single column vertically */
   width: 100%;
   margin: 8px;
+  min-height: 130px;
```

Comment:
why is this here? kind of a weird thing to hardcode, I would like to avoid that

Reply:
With minwidth: 
![image](https://github.com/PrairieLearn/PrairieLearn/assets/16738261/28a430e7-6c83-4099-a181-921b60632cdd)


Without min-width:
![image](https://github.com/PrairieLearn/PrairieLearn/assets/16738261/e1ab0bbe-1351-4e33-8f3c-38a2ccb34017)

When I was testing, dragging between boxes that were resizing and and starting out with a 0-width construction area felt pretty annoying. I wanted it to feel more like the traditional order-blocks where the height of the columns is set based on the tallest column. The main thing I'd like to avoid is resizing while the user is dragging so perhaps dropzone height should be set as the problem's tallest block?

Reply:
oh I see. Ya it is nice to have it start with some height. 

I don't think resizing during dragging is necessarily a problem though, that's what the behavior is now. A few options are starting same height as smallest or biggest block, or just starting as same size as the starting zone. I'm not sure what's best without playing with it a bit. 

We can always just ship something and then tweak it later if we feel like.

Reply:
When stacking vertically I don't think its as much of a problem. However it felt a bit clunky to me when dragging things around horizontally and having it dynamically resize vertically. I'll go with setting it to the biggest block for right now and see what you think.

Reply:
A size with `rem` units may be a better option here.

Reply:
updated!